doc/devel/development.md

```diff
@@ -0,0 +1,1946 @@
+| `service`             | `service`            |                                                                                                                                      |
+| `service_path`        | `subservice`         |                                                                                                                                      |
+| `entity_name`         | `name`               |                                                                                                                                      |
+| `entity_type`         | `type`               |                                                                                                                                      |
+| `timezone`            | `timezone`           |                                                                                                                                      |
+| `timestamp`           | `timestamp`          |                                                                                                                                      |
+| `apikey`              | `apikey`             |                                                                                                                                      |
+| `endpoint`            | `endpoint`           |                                                                                                                                      |
+| `protocol`            | `protocol`           | Name of the device protocol, for its use with an IoT Manager. IE: IoTA-UL                                                            |
+| `transport`           | `transport`          |                                                                                                                                      |
+| `attributes`          | `active`             |                                                                                                                                      |
+| `lazy`                | `lazy`               |                                                                                                                                      |
+| `commands`            | `commands`           |                                                                                                                                      |
+| `internal_attributes` | `internalAttributes` | List of internal attributes with free format for specific IoT Agent configuration. I.E:LWM2M mappings from object URIs to attributes |
+| `static_attributes`   | `staticAttributes`   |                                                                                                                                      |
+| `expressionLanguage`  | `expresionLanguage`  | _Removed_                                                                                                                            |
```

Comment:
```suggestion
```

Idem

Reply:
In this case, it is better to do in a subsequent PR. Noted here: https://github.com/telefonicaid/iotagent-node-lib/issues/1329#issue-1643605738
NTC